README.md

```diff
@@ -0,0 +1,489 @@
+# sanctuary-def
+
+sanctuary-def is a run-time type system for JavaScript. It facilitates the
+definition of curried JavaScript functions which are explicit about the number
+of arguments to which they may be applied and the types of those arguments.
+
+It is conventional to import the package as `$`:
+
+```javascript
+const $ = require('sanctuary-def');
+```
+
+The next step is to apply `$.create` to a list of types (an environment):
+
+```javascript
+const def = $.create($.env);
+```
+
```

Comment:
You have included a list of all the types that are part of `$.env` further down the page (at which point there is actually no mention of `$.env`). I would prefer to see the types enumerated here, since one of my first questions after reading this sentence was "What types are included in `$.env`?".

If the list of types defined in `$.env` becomes to large, it may be appropriate to link to a full list of types further down the page or in a separate page, rather than trying to list them all right here.